Nokia/drg/res/values/config.xml

```diff
@@ -0,0 +1,117 @@
+    <dimen name="navigation_bar_width">126.0px</dimen>
+    <dimen name="notification_header_padding_top">10.0dip</dimen>
+    <dimen name="notification_header_padding_bottom">11.0dip</dimen>
+    <dimen name="status_bar_height_portrait">89.0px</dimen>
+    <dimen name="status_bar_height">89.0px</dimen>
+    <dimen name="status_bar_height_landscape">89.0px</dimen>
+
+	<integer name="config_screenBrightnessSettingMinimum">1</integer>
+    <integer name="config_screenBrightnessDoze">17</integer>
+    <integer name="config_screenBrightnessSettingDefault">66</integer>
+    <integer name="config_autoBrightnessBrighteningLightDebounce">1000</integer>
+    <integer name="config_autoBrightnessDarkeningLightDebounce">1000</integer>
+    <integer name="config_screenBrightnessDark">1</integer>
+    <integer name="config_screenBrightnessDim">10</integer>
+    <integer name="config_brightness_ramp_rate_fast">180</integer>
+    <integer name="config_brightness_ramp_rate_slow">60</integer>
+    <fraction name="config_autoBrightnessAdjustmentMaxGamma">299.99695%</fraction>
+
+    <integer-array name="config_autoBrightnessLevels">
+        <item>1</item>
+        <item>2</item>
+        <item>4</item>
+        <item>5</item>
+        <item>8</item>
+        <item>13</item>
+        <item>17</item>
+        <item>21</item>
+        <item>26</item>
+        <item>30</item>
+        <item>34</item>
+        <item>39</item>
+        <item>60</item>
+        <item>140</item>
+        <item>310</item>
+        <item>400</item>
+        <item>500</item>
+        <item>600</item>
+        <item>1000</item>
+        <item>1200</item>
+        <item>1500</item>
+        <item>3000</item>
+        <item>3500</item>
+        <item>4000</item>
+    </integer-array>
+    <integer-array name="config_autoBrightnessLcdBacklightValues">
+        <item>2</item>
+        <item>2</item>
+        <item>3</item>
+        <item>3</item>
+        <item>8</item>
+        <item>10</item>
+        <item>12</item>
+        <item>15</item>
+        <item>17</item>
+        <item>24</item>
+        <item>30</item>
+        <item>30</item>
+        <item>44</item>
+        <item>45</item>
+        <item>48</item>
+        <item>55</item>
+        <item>64</item>
+        <item>66</item>
+        <item>69</item>
+        <item>84</item>
+        <item>93</item>
+        <item>105</item>
+        <item>200</item>
+        <item>240</item>
+        <item>255</item>
+    </integer-array>
+
+	<integer-array name="config_virtualKeyVibePattern">
+        <item>50</item>
+    </integer-array>
+
+	<integer-array name="config_fingerprintSuccessPattern">
+        <item>0</item>
+        <item>75</item>
+    </integer-array>
+    <integer-array name="config_fingerprintFailPattern">
+        <item>0</item>
+        <item>75</item>
+        <item>100</item>
+        <item>75</item>
+    </integer-array>
+
+    <bool name="config_enableCpuBoostForTouch">true</bool>

```

Comment:
Doesn't exist in AOSP.
If other ROMs uses it, please point it to me, I can accept to merge than then